### ISSUE TITLE
Request: Improve TTBG' AMO page description

### ISSUE THREAD

**Issue by EstherMoellman** [jingyu9575/tabs-to-bookmark-groups#16]
Hi @jingyu9575 , this is not a priority request. But I believe is a request good to have in mind.

Devs or advanced users usually don't care about add-on descriptions, they are pragmatic, they like to test add-ons regardless descriptions, images, titles, reviews, fashions etc.

However, in real world most of the users they don't use add-ons at all, because they don't know what an add-on is.
And the minority using add-ons, they also are far away of being experts, most of them just use add-ons because somebody else uses, or just because is "recommended", mass behavior or similar.
I'm trying to say that a good TTBG' AMO page description may be useful for most of the users.

Currently TTBG has a very generalist description AMO' page, more than enough for Devs or advanced users.
However IMO, average users will need more explanations. Also it will be nicer and more friendly looking, if the TTBG' black icon (with the star inside) is changed with a full colored icon (blue or orange etc), and with updated images (including images of options page, sidebar, popup etc).

As an structure of details, I suggest a TTBG' description of:
1) Purpose
2) Functions
3) Features
4) Functions and features currently under development, or for future development
5) Alerts, notifications or advice
6) Compared advantages against similar add-ons
7) General info (author, open source etc)

I never will say what you must write, because this is something very personal, very yours.
But at my TTBG' review in AMO' page you will find what kind of explanation I believe as useful.
[Also here](https://www.ghacks.net/2020/02/22/simple-tab-groups-is-a-firefox-extension-for-organizing-your-tabs/) at this Ghacks' article, below at "comments area", you will find more explanations I gave to users.

**Comment by jingyu9575** [comment]
Yes I know the description is unhelpful, but this extension is still under development and lacks many features. I'd keep the description short for now, to avoid disappointing users and getting low-star reviews. I think it is better to rewrite it after most features are completed.
